docs/menus/pause.json

```diff
@@ -1,17 +1,38 @@
 {
+  "stationMenus": [
+    {
+      "label": "Shakes",
+      "note": "Mixins: M&Ms, Oreos, Reese’s Peanut Butter Cups"
+    },
+    {
+      "label": "Soft-Serve",
+      "note": "Hand-swirled in a cup or a cone",
+    },
+    {
+      "label": "Pizza",
+      "note": "Includes a cup of Marinara, Ranch, BBQ, or Honey Mustard sauce"
```

Comment:
On another note, large orders are a pain-point for the pause. We could use the note here to communicate "For orders of 4 or more pizzas, visit Oleville"